plugin/jumpmotion.vim

```diff
@@ -248,7 +248,7 @@ function JumpMotion(...) abort range
         endfor
 
         try
-          keepjumps execute 'normal! ' . edit
+          noautocmd keepjumps execute 'normal! ' . edit
```

Comment:
Could you remove the dot? Just now look that it is the only place where I used it.
```suggestion
          noautocmd keepjumps execute 'normal!' edit
```